importer/test_data/import_after_usage.expected.asm

```diff
@@ -0,0 +1,7 @@
+machine Foo {
+}
+mod module {
+    machine Bar {
+        Foo foo;
```

Comment:
Unrelated to this PR, but as is, this would not compile, would it, because `Foo` is not available like that. Maybe we should print (and also parse) that as `::Foo`? Or should we just live with the fact that it doesn't work as input?

Reply:
I'm not sure I understand. At the point of that Display trait, we are already in a situation where everything is inlined, right? So even if we had crates, they would not really exist, everything is turned into a single file.

So I guess the question is if we want to have a special symbol name for the top level "module" (or meta-module, meaning the module containing all top-level module) that is available everywhere or just have a special syntax like `::...`.

I kind of would like to have the ideal that everything we display should also be parseable, even though I know that it migth not always be achievable. But at least we should introduce something that distinguishes relative from absolute paths in the Display trait to make it harder to trip over such things.

Reply:
```
machine Machine {}
mod module {
   use ::Machine;
}
```

Modules are compilation units. Without absolute paths, you can compile `module` as a standalone thing. With absolute paths, you cannot. Not the end of the world and probably still worth introducing absolute paths as I agree with the idea that we should be able to parse everything, but still a bit odd to me. If you make "crates" the compilation unit, you don't have that problem because you drop the idea that you can compile each module, and always start from the root of the crate.